maven-bom/pom.xml

```diff
@@ -18,14 +17,13 @@ KIND, either express or implied.  See the License for the
 specific language governing permissions and limitations
 under the License.
 -->
-
 <project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 https://maven.apache.org/xsd/maven-4.0.0.xsd">
   <modelVersion>4.0.0</modelVersion>
   <parent>
     <groupId>org.apache.maven</groupId>
     <artifactId>maven-parent</artifactId>
-    <version>35</version>
-    <relativePath />
+    <version>38</version>
+    <relativePath/>
```

Comment:
Any reason why the BOM does not have the following has a parent ?
```
    <groupId>org.apache.maven</groupId>
    <artifactId>maven</artifactId>
    <version>4.0.0-alpha-4-SNAPSHOT</version>
```

Reply:
If the problem is that the parent contains too much stuff, I still think it would be better to use the parent, and maybe create another pom to contain all the definitions used at build time and have other modules inherit from that one.

Reply:
Chicken and egg problem. We went through this with Robert.